packages/plugin-ext/src/hosted/node/plugin-service.ts

```diff
@@ -78,37 +75,32 @@ export class HostedPluginServerImpl implements HostedPluginServer {
     }
 
     // need to run a new node instance with plugin-host for all plugins
-    deployFrontendPlugins(frontendPlugins: PluginDeployerEntry[]): Promise<void> {
-        // get metadata
-        frontendPlugins.forEach(frontendPluginDeployerEntry => {
-            const pluginMetadata = this.reader.getPluginMetadata(frontendPluginDeployerEntry.path());
-            if (pluginMetadata) {
-                this.currentFrontendPluginsMetadata.push(pluginMetadata);
-                this.logger.info('HostedPluginServerImpl/ asking to deploy the frontend Plugin', frontendPluginDeployerEntry.path(), 'and model is', pluginMetadata.model);
+    async deployFrontendPlugins(frontendPlugins: PluginDeployerEntry[]): Promise<void> {
+        for (const plugin of frontendPlugins) {
+            const metadata = await this.reader.getPluginMetadata(plugin.path());
+            if (metadata) {
+                this.currentFrontendPluginsMetadata.push(metadata);
+                console.info(`Deploying frontend plugin "${metadata.model.name}@${metadata.model.version}" from "${metadata.model.entryPoint.frontend || plugin.path()}"`);
```

Comment:
why console instead of logger ?

Reply:
We've started with `Logger` and fought to use a primary API, but then learned that we cannot force all 3rd party libraries to use `Logger`, so we hooked up `console` to use `Logger` always. You can use `Logger` but it is just more code.

Reply:
pls keep logger, it's avoid to see if ppl miss to use console just for printing their debug output.

Reply:
>  it's avoid to see if ppl miss to use console just for printing their debug output.

Don't we have code reviews to detect it?

Let's discuss it tomorrow at dev meeting. We have [coding rules](https://github.com/theia-ide/theia/wiki/Coding-Guidelines) for the project. It should be agreed by everybody to avoid arguing over them during PRs. If something is not there it's up to everybody.

Reply:
I've touched all these lines anyway.

Reply:
so, please avoid to do big changes, just the fix or separate commits